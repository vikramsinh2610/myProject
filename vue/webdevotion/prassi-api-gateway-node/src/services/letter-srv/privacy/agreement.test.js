const Agreement = require('./agreement');

describe('Agreement', () => {
  test('should have these properties', () => {
    const agreement = new Agreement({
      name: 'Mickey Mouse is nice',
      content: 'This is a very long story...',
      agree: true,
    });
    expect(agreement).toHaveProperty('name');
    expect(agreement).toHaveProperty('content');
    expect(agreement).toHaveProperty('agree');
  });

  test('should return it s JSON Schema', () => {
    expect(Agreement.getJSONSchema()).toMatchSnapshot();
  });
});
