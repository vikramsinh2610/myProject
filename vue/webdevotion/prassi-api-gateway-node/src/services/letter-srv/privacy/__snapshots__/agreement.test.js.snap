// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Agreement should return it s JSON Schema 1`] = `
Object {
  "properties": Object {
    "agree": Object {
      "type": "boolean",
    },
    "content": Object {
      "type": "string",
    },
    "name": Object {
      "type": "string",
    },
  },
  "required": Array [
    "name",
    "content",
    "agree",
  ],
  "type": "object",
}
`;
